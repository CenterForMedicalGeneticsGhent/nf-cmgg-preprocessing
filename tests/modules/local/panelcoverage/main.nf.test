nextflow_process {

    name "Test Process PANELCOVERAGE"
    script "modules/local/panelcoverage/main.nf"
    process "PANELCOVERAGE"

    test("Should run without failures") {

        when {
            params {

            }
            process {
                """
                // define inputs of the process here. Example:
                input[0] = [[id:'test', single_end:false], file("assets/testdata/test-depth-per-base.bed.gz", checkIfExists: true)]
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
