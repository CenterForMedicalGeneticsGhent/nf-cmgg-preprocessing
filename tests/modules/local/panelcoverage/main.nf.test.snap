{
    "Should run without failures": {
        "content": [
            {
                "versions": [
                    "versions.yml:md5,cdc869a829c25bc6ddf9068f63941798"
                ],
                "regions_txt": [
                    [
                        {
                            "id": "test",
                            "single_end": false
                        },
                        "test.region.dist.txt:md5,c1163eb5af9994f92a07dc3ba827cccc"
                    ]
                ]
            }
        ],
        "timestamp": "2022-12-12T07:39:01+0000"
    }
}